ci/run_unit_tests.sh

```diff
@@ -8,7 +8,7 @@ if [ $DB = sqlite ]; then
     extra_flags='-n auto'
 fi
 
-py.test $extra_flags \
+py.test --cov=hil $extra_flags \
```

Comment:
This is going to miss a bunch of stuff, since we have other scripts that run integration tests with e.g. keystone, apache, obmd.

Reply:
true, but for some reason, when I add the `--cov=hil` to all the `py.test`, the coverage report returns a 0.0%. I suspect it is expecting us to have a one `py.test` to run all the tests, at one time. Still investigating.